### DISCUSSION TITLE
[VSCodium Bin] Can't authenticate ("You don't have access" despite having access)

### DISCUSSION THREAD

**Discussion by hacatu** [community/community#6641] [Copilot Conversations]
I got an email from github today that I was now off the waitlist and could use the copilot preview.  I downloaded the vsix file for the extension and installed it in VSCodium.  It then prompted me to give it a personal access token, so I generated a new one on github and entered it.  Then it told me "You don't have access to GitHub Copilot.  Join the waitlist."  But I do have access, I got the email earlier today.  The access token I gave it had the permissions `public_repo, read:packages, read:public_key, read:user, user:email`.  I don't think it should need any permissions but I couldn't find any documentation on what permission to give the PAT and that seems like what's most likely causing me to be unable to use copilot.  VSCodium lists github under my logged in accounts so I know the PAT kind of worked.  Has anyone been able to get the preview working on VSCodium or does anyone know why I can't use the preview even after getting the email?

**Comment by Binary-Eater** [comment]
I wonder if Copilot API/internals only accepts tokens generated by https://vscode-auth.github.com/authorize/ (if so, that would explain why the PAT doesn't work).

**Comment by hsenag** [comment]
At least for now, please try using VSCode releases from https://code.visualstudio.com/ to use Copilot.

**Reply by fee1-dead** [reply]
Hello, I am on void linux with musl libc. I cannot use binary releases. Could you please support open source builds? If not I would have to use NeoVim.
